### DISCUSSION TITLE
Modify the source code locally and find that the code does not take effect after reinstalling.

### DISCUSSION THREAD

**Discussion by D0n9** [gildas-lormeau/SingleFile#1394] [Q&A] [answered]
How to do local development and debugging?
After repeated testing, it was found that modifying the files in the src/ folder did not take effect.

**Comment by gildas-lormeau** [comment]
You have to run `npm run dev` after doing the changes in the file.

**Reply by D0n9** [reply]
Thank you for your reply. An error occurred when running **npm running dev**, is the file missing?

```
> single-file@1.1.40 dev


> npx rollup -c rollup.config.dev.js


single-file-core/single-file.js → lib/single-file.js...
[!] Error: Could not resolve entry module (single-file-core/single-file.js).
```

**Reply by gildas-lormeau** [reply, marked as answer]
You also have to fork [`single-file-core`](https://github.com/gildas-lormeau/single-file-core) into the parent folder of SingleFile on your machine.
